### PR TITLE
Say goodbye to dep, welcome modules

### LINKED ISSUE #42: Switch to go modules
https://blog.golang.org/modules2019
> Our aim is for Go 1.13, scheduled for August 2019, to enable module mode by default (that is, to change the default from auto to on) and deprecate GOPATH mode. In order to do that, we’ve been working on better tooling support along with better support for the open-source module ecosystem.

It's about time to use go modules.

### PR DESCRIPTION
**Pull request by JohnTitor**
Fixes #42 

dep did a great job, we say goodbye to him.
And we welcome Go Modules.

I don't know much about CircleCI, please review.

r? @yaminoma

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -3,22 +3,16 @@ jobs:
   build:
     docker:
       - image: circleci/golang:1.11.5
-    working_directory: /go/src/github.com/student-kyushu/frau
```
Comment (yaminoma) [inline .circleci/config.yml]: What is the reason for erasing this line?

File: .circleci/config.yml [hunk 2]
```diff
@@ -3,22 +3,16 @@ jobs:
...
+        environment:
+          GO111MODULE: "ON"
     steps:
       - checkout
       - restore_cache:
           keys:
-            - vendor-{{ checksum "Gopkg.lock" }}
-            - dep
+            - go-mod-v1-{{ checksum "go.sum" }}
```
Comment (yaminoma) [inline .circleci/config.yml]: [imo] I often write cache keys in the style below.
`go-mod-{{ checksum "go.sum" }}/1`

File: .gitignore [hunk 3]
```diff
@@ -4,6 +4,7 @@ vendor/
 # Exec binary
 frau
 frau_*
+frau.*
```
Comment (yaminoma) [inline .gitignore]: What about putting it together?

File: .travis.yml [hunk 4]
```diff
@@ -1,24 +1,22 @@
 language: go
 
 go:
-    - "1.11"
-    - master
+  - "1.11.x"
```
Comment (yaminoma) [inline .travis.yml]: You should also specify a minor version.

File: .circleci/config.yml (hunk 1 above)
Comment (JohnTitor) [reply .circleci/config.yml]: This is a mistake, I fixed.

File: .circleci/config.yml (hunk 2 above)
Comment (JohnTitor) [reply .circleci/config.yml]: I adopted your suggestion, thank you.

File: .gitignore (hunk 3 above)
Comment (JohnTitor) [reply .gitignore]: I brought them together, okay?

File: .travis.yml (hunk 4 above)
Comment (JohnTitor) [reply .travis.yml]: Hmm, `1.11.x` is used as a latest patch version(i.e. 1.11.5), I think.  Is it better to specify a version, or is my understanding wrong?

Comment (yaminoma) [reply .travis.yml]: You should avoid using latest.
You should do it manually if the minor version goes up.

Comment (JohnTitor) [reply .travis.yml]: I got it, fixed.

**Comment by yaminoma** [comment]
@KoujiroFrau r+

**Comment by KoujiroFrau** [comment]
:pushpin: Commit 409bcaf82790bfb7150ed112aafb9bd975fa5dae has been approved by `yaminoma`

**Comment by KoujiroFrau** [comment]
:postbox: This pull request is queued. Please await the time.

**Comment by JohnTitor** [comment]
@KoujiroFrau r-

**Comment by KoujiroFrau** [comment]
:outbox_tray: This has been cancelled from the approved queue by `JohnTitor`

**Comment by JohnTitor** [comment]
@KoujiroFrau r=yaminoma

**Comment by KoujiroFrau** [comment]
:pushpin: Commit 409bcaf82790bfb7150ed112aafb9bd975fa5dae has been approved by `yaminoma`

**Comment by KoujiroFrau** [comment]
:postbox: This pull request is queued. Please await the time.

**Comment by JohnTitor** [comment]
@KoujiroFrau r+

**Comment by KoujiroFrau** [comment]
:pushpin: Commit 409bcaf82790bfb7150ed112aafb9bd975fa5dae has been approved by `JohnTitor`

**Comment by KoujiroFrau** [comment]
:postbox: This pull request is queued. Please await the time.

**Pull request merged**